compose.override.yaml

```diff
@@ -2,5 +2,5 @@ services:
 ###> doctrine/doctrine-bundle ###
   database:
     ports:
-      - "5432:5432"
+      - "5432"
```

Comment:
This allows to have several projects opened at the same time.

Reply:
And this is solved by Symfony CLI
![image](https://github.com/user-attachments/assets/0fed9f0b-1ed3-48cd-95c3-d3633494c504)